service/javadsl/api/src/test/java/com/lightbend/lagom/api/mock/DefaultMethodWithParamsService.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.lightbend.lagom.api.mock;
+
+import com.lightbend.lagom.javadsl.api.Descriptor;
+import com.lightbend.lagom.javadsl.api.Service;
+import com.lightbend.lagom.javadsl.api.ServiceCall;
+import com.lightbend.lagom.javadsl.api.transport.Method;
+
+import java.util.UUID;
+
+import static com.lightbend.lagom.javadsl.api.Service.named;
+import static com.lightbend.lagom.javadsl.api.Service.restCall;
+
+public interface DefaultMethodWithParamsService extends Service {
+
+  ServiceCall<UUID, String> hello(String name);
+
+  default String serviceName(String param) {
+    return param;
+  }
```

Comment:
I would like to understand the modification to have such a method. Is it the goal to do some transformations on the service name? It looks that's the case.

Reply:
> Is it the goal to do some transformations on the service name?

It's can be too. But our case is the transformation of routes (add another one).
The issue related to using our library [Lagom OpenAPI](https://github.com/taymyr/lagom-openapi).
We will plan to migrate to _Additional Routes_ on the next versions, but this library was written for Lagom 1.4 and use the default method to add the route for OpenAPI specification. See [OpenAPIService.kt](https://github.com/taymyr/lagom-openapi/blob/master/java/api/src/main/kotlin/org/taymyr/lagom/javadsl/openapi/OpenAPIService.kt).
The real case you can see in one of our samples https://github.com/taymyr/lagom-samples/tree/master/openapi/java

PS: @gnuzzz is my colleague and we caught a problem when migrating our service to JVM 11.